docs/GoogleSheets-batchsink.md

```diff
@@ -56,14 +56,17 @@ Service account json can be generated on Google Cloud
 * **File Path**: Path on the local file system of the service account key used for
   authorization. Can be set to 'auto-detect' when running on a Dataproc cluster.
   When running on other clusters, the file must be present on every node in the cluster.
-  When using 'auto-detect', the cluster needs to be configured with the scope required by the Google Drive Api,
-  otherwise the preview as well as pipeline run will fail with insufficient permission error. Here, the required scope 
-  is "https://www.googleapis.com/auth/drive".
+  
+  When set to 'auto-detect', the cluster needs to be configured with the scope required by the Google Drive and Sheets Api,
+  otherwise the preview as well as pipeline run will fail with insufficient permission error.
```

Comment:
@itsankit-google Change to: "Otherwise, the preview and pipeline run will fail with an insufficient permission error."